### ISSUE TITLE
Request - new citation type: referenced

### ISSUE THREAD

**Issue by KatherineLAnderson** [ArctosDB/arctos#8139]
Help us understand your request (check below):
- [ ] search or data request (help with SQL or provide the specific fields you are trying to retrieve for your collection)
- [ ] add a new Github user to the Arctos Users team
- [ ] a new barcode series
- [x] a new term for data entry or management 
- [ ] a change or new report or label
- [ ] update to an Agent (split, merge, or other type of change)
- [ ] bulkload changes (to Agents, Identifiers, etc that are not already available as a bulkloading tool)
- [ ] new feature or otherwise a good idea to make Arctos even better
- [ ] quarantine taxon name and suggested replacement (please supply links)
- [ ] other

**Describe what you're trying to do**

**Comment by KatherineLAnderson** [comment]
> I don't understand how this isn't a duplicate of https://arctos.database.museum/info/ctDocumentation.cfm?table=CTCITATION_TYPE_STATUS#voucher 

I think I had an adverse reaction to "voucher" because we don't use that word as much in paleo as they do in biological collections, but it would work according to the definition in the code table. I just hope it isn't misleading for paleo users. I can move forward with this unless any other paleo folks want to chime in on this. @WaigePilson @Nicole-Ridgwell-NMMNHS 

> or https://arctos.database.museum/info/ctDocumentation.cfm?table=CTCITATION_TYPE_STATUS#referral

This one specifies non peer reviewed literature which is why it doesn't work.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I agree with @KatherineLAnderson that the term voucher isn't used much in paleontology, although I have been using it for our un-figured peer reviewed citations per Arctos' definition. I think it would be beneficial to review how Arctos's definition lines up with how the term is actually used and defined. Doing some quick google searches, the definition I see most frequently is something like "Vouchers are representative specimens of a species that are physically deposited in a collection or a curated museum or institution and are accompanied by metadata and (optionally) DNA samples." (from Turney et al. 2015 http://dx.doi.org/10.7717/peerj.1168), although I'd be happy to be corrected by folks working on modern organisms who use the term more frequently.

**Comment by dustymc** [comment]
The definition of the word outside of Arctos isn't terribly important, only the Arctos definition should guide usage - but of course everyone's going to be happier when we can follow Standards or common usage.

> as they do in biological collections

Yea, that's come up a couple times, people confuse themselves there too. That usage (which varies wildly among NH collections) was **_not_** in the intention of the term, the term was just a way of expanding the link between "something in a museum" (no "voucher" necessary!) and literature beyond nomenclature/taxonomy.

I have no attachment to the term, but it does need to play nice with the other terms (which are mostly from various Codes so less-easy to change) - eg a past tense would be awkward.

I also have no attachment to the separation of voucher and referral - I'm sure I objected, that information is carried by the publication and isn't necessary here - but clearly someone thought it was important. There's no arbitrary usage or overlap so it's not wrong or evil, just doesn't seem like a necessary split.

**Comment by WaigePilson** [comment]
We actually do use voucher in paleobotany collections--researchers collecting large samples (many thousands of specimens) will often conduct a census (noting taxa, maybe taking pictures) in the field but only collect exemplary/type/novel/interesting specimens as "vouchers".

I think most frequently I'm linking specimens to a publication as type or figured (apparently figured isn't an accepted term in this code table?? Maybe there's someplace else figured specimens are noted??). [As an aside, the other term paleobotanists use is "exemplar" (because in paleobotany often we're assigning specimens to non-Linnean operational taxonomic units aka morphotypes, so "type" is inappropriate); I probably will just add these as a remark somewhere as I doubt that adding "exemplar" to this code table would get approved.]

[As another aside: I'd argue that the Arctos definition must meet the community definition. If there are terms used in Arctos differently than the community uses them, we are set up for failure]

I would vote to use "voucher" for non-type specimens rather than introducing a new term (I can use a remark to note if it's elevated to the level of "exemplar", figured, just referenced, etc). For paleobotany, I think this is the best solution.

**Comment by campmlc** [comment]
I think it that voucher should be maintained for citations of actual physical objects deposited in the collection (as ""Vouchers are representative specimens of a species that are physically deposited in a collection or a curated museum or institution and are accompanied by metadata and (optionally) DNA samples." (from Turney et al. 2015 http://dx.doi.org/10.7717/peerj.1168) 
Is that consistent with the use of "cited" or "referenced" or "exemplar" in this case? If not, then I suggest we add a different term or alter the definition or name of "referral".

**Comment by KatherineLAnderson** [comment]
The easiest thing for me to do is ask for no change and use voucher--which, by definition in the code table (and community standards--depending on your community, I suppose), is what I am trying to indicate. If that's what Nicole is already doing, and what Paige will do, then I am good with it. We will work on using the lexicon per Arctos here!

@WaigePilson  re: Figured specimens that are not types, I used "basis of illustration". CT Definition: "figured in a drawing, photograph, or other representation. Does not apply to publications in which the specimen was also designated a taxonomic type status."

**Comment by dustymc** [comment]
>figured

https://arctos.database.museum/info/ctDocumentation.cfm?table=CTCITATION_TYPE_STATUS#basis_of_illustration (which may push "evil" pretty hard, I'm relatively sure it and 'voucher' are arbitrarily used, see also https://github.com/ArctosDB/arctos/issues/1861)

>would get approved

FWIW my take is kinda always

1. overlaps some existing term, can't be functionally defined, isn't appropriate for the node====>👎
2. doesn't overlap other stuff, can be functionally defined, structurally appropriate=====>👍


>If there are terms used in Arctos differently than the community uses them, we are set up for failure

Absolutely, but as long as "thing" is defined as "whatever" in Arctos, then that is the **only** proper usage within Arctos. If everyone else thinks "thing" means "not whatever" then PLEASE file an issue to make it better. ('voucher' was created from a much smaller world - I believe Arctos was one collection at the time! - and it's not surprising to find that the larger world sees things differently.)

> voucher should be maintained for citations of actual physical objects 

Unless you want to go check them all, no, there's no realistic way we're changing that. (We can easily change the term, but losing the 'vague literature/record link' type would be a huge - 404491 of them at the moment! - PITA.)

> add a different term or alter the definition

See above, as long as there's a clear separation and a migration path that's probably OK.

>or name

That'll always be the easy part from here, it's the **_concept_** that matters.

**Comment by dustymc** [comment]
Resolved?
